system/jlib/jdebug.cpp

```diff
@@ -1270,9 +1330,9 @@ void getCpuInfo(unsigned &numCPUs, unsigned &CPUSpeed)
         numCPUs = 1;
 
     // max cpu freq (KHz) may be in:
-    // /sys/devices/system/cpu/cpu[0-X]/cpufreq/cpuinfo_max_freq
+    // /sys/devices/system/cpu/cpu[0-X]/cpufreq/systemprocessinfo_max_freq
```

Comment:
suspect this is an accidental s&r change too?

Reply:
Also, fwiw, I don't have a /sys/devices/system/cpu/cpu*/cpufreq/ directory in either my host (wsl2), or DD container contexts, maybe native would...

I do however have an [empty] /sys/devices/system/cpu/cpufreq directory in both contexts.